### PR TITLE
feat: Upgrade knative to latest knative-serving

### PR DESCRIPTION
**Pull request by adityajariwala**
**What problem does this PR solve?**:
Update knative to most recent net-istio supported version of knative

remove knative-eventing as only serving is used

Made change such that to update template, only bumping version in serving/Chart.yaml is needed

**Which issue(s) does this PR fix?**:

https://d2iq.atlassian.net/browse/D2IQ-96638

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

**Comment by adityajariwala** [comment]
Waiting on [chart to be merged](https://github.com/mesosphere/charts/pull/1406)

**Comment by adityajariwala** [comment]
`kommander-applications/pre-commit ` test failure is due to one commit that is somehow still failing name length checks, even though I squashed that commit. It doesn't show up here, but somehow when checking out in TC it does. Not sure how to fix that

File: common/kommander-operator/helmrelease.yaml
Comment (takirala) [inline common/kommander-operator/helmrelease.yaml:15]: ??

Comment (adityajariwala) [reply common/kommander-operator/helmrelease.yaml:15]: Stupid auto-refactor in GoLand @takirala

**Review by takirala — COMMENTED** [review]
LGTM pending `common/helm-repositories` changes pointing to the fork. We can revert and rerun after merging the charts  repo PR.

**Review by takirala — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

**Comment by adityajariwala** [comment]
@d2iq-mergebot test kommander-applications/upgrade-e2e-tests

**Pull request merged**
